src/main/java/frc/robot/Runtime.java

```diff
@@ -0,0 +1,235 @@
+			send(Auto.climbPad(this.robot.drivebase, pitch,
+				Constants.AUTO_PAD_ENGAGE_VELOCITY, Constants.AUTO_PAD_INCLINE_VELOCITY), "Commands/Climb Pad"));
+		AutonomousTrigger.OnTrue(new InstantCommand(()->this.auto.getSelected().schedule()));
+	}
+	@Override
+	public void robotPeriodic() {
+		CommandScheduler.getInstance().run();
+	}
+
+	@Override
+	public void disabledInit() {}
+	@Override
+	public void disabledPeriodic() {}
+	@Override
+	public void disabledExit() {}
+
+	@Override
+	public void autonomousInit() {}
+	@Override
+	public void autonomousPeriodic() {}
+	@Override
+	public void autonomousExit() {}
+
+	@Override
+	public void teleopInit() {}
+	@Override
+	public void teleopPeriodic() {}
+	@Override
+	public void teleopExit() {}
+
+	@Override
+	public void testInit() {}
+	@Override
+	public void testPeriodic() {}
+	@Override
+	public void testExit() {}
+
+
+
+
+
+	private void setupXbox(InputDevice... inputs) {
+		System.out.println("Initializing Xbox Control Scheme.");
+		InputDevice controller = inputs[0];
+		TeleopTrigger.OnTrue(
+			this.robot.drivebase.tankDriveVelocity(
+				Xbox.Analog.LY.getDriveInputSupplier(controller,
+					Constants.DRIVE_INPUT_DEADZONE, Constants.DRIVE_INPUT_VEL_SCALE, Constants.DRIVE_INPUT_EXP_POWER),
+				Xbox.Analog.RY.getDriveInputSupplier(controller,
+					Constants.DRIVE_INPUT_DEADZONE, Constants.DRIVE_INPUT_VEL_SCALE, Constants.DRIVE_INPUT_EXP_POWER)
+			)
+		);
+		EnabledTrigger.OnTrue(new Vision.CameraControl(
+			controller.button(Xbox.Digital.A.value, CommandScheduler.getInstance().getDefaultButtonLoop()).rising(),
+			controller.button(Xbox.Digital.B.value, CommandScheduler.getInstance().getDefaultButtonLoop()).rising()
+			// Xbox.Digital.A.getCallbackFrom(controller)
+		));
+	}
+	// private void setupTestXbox(InputDevice... inputs) {
+	// 	System.out.println("Initializing Xbox Test Control Scheme.");
+	// 	TeleopTrigger.OnTrue(
+	// 		this.robot.drivebase.tankDriveVelocity(
+	// 			Xbox.Analog.LY.getDriveInputSupplier(inputs[0],
+	// 				Constants.DRIVE_INPUT_DEADZONE, Constants.DRIVE_INPUT_VEL_SCALE, Constants.DRIVE_INPUT_EXP_POWER),
+	// 			Xbox.Analog.LY.getDriveInputSupplier(inputs[1],
+	// 				Constants.DRIVE_INPUT_DEADZONE, Constants.DRIVE_INPUT_VEL_SCALE, Constants.DRIVE_INPUT_EXP_POWER)
+	// 		)
+	// 	);
+	// }
+
+
+
+
+
+	private void setupArcade(InputDevice... inputs) {
+		System.out.println("Initializing Arcade Board Control Scheme.");
+		InputDevice		// aliases for less confus
+			lstick = inputs[0],
+			rstick = inputs[1]
+		;
+		TeleopTrigger.OnTrue(
+			// this.robot.drivebase.tankDriveVelocity(
+			// 	Attack3.Analog.Y.getDriveInputSupplier(lstick,
+			// 		Constants.DRIVE_INPUT_DEADZONE, Constants.DRIVE_INPUT_VEL_SCALE, Constants.DRIVE_INPUT_EXP_POWER),
+			// 	Attack3.Analog.Y.getDriveInputSupplier(rstick,
+			// 		Constants.DRIVE_INPUT_DEADZONE, Constants.DRIVE_INPUT_VEL_SCALE, Constants.DRIVE_INPUT_EXP_POWER)
+			// )
+			this.robot.drivebase.tankDrivePercent(
```

Comment:
Weren't you planning to use the velocity drive?

Reply:
Oh,  this is for arcade drive...

Reply:
Yes I updated the controls in the latest commit (a few mins ago)

Reply:
Actually... I'm confused... this is tank drive.  What is the arcade board?  Is that the thing with the two flight sticks and the button box with the switches on top?

Reply:
"Arcade Board" is just the 2 sticks, "Control Board" is that plus the buttonbox, and "Comp controls" are the 2 arcade sticks plus and xbox controller because the buttonbox was causing issues and that was the best thing I could come up with for tommorrow.